packages/react-component-library/package.json

```diff
@@ -26,14 +26,12 @@
     "directory": "packages/react-component-library"
   },
   "scripts": {
-    "analyze": "webpack --mode production --config=webpack/analyze.js",
+    "analyze": "webpack --config=webpack/analyze.js",
     "build": "run-s build:es build:cjs types",
-    "build:cjs": "BABEL_ENV=cjs webpack --mode production --config=webpack/prod.js",
+    "build:cjs": "BABEL_ENV=cjs webpack --config=webpack/prod.js",
     "build:es": "BABEL_ENV=es babel src --out-dir \"dist/es\" --extensions \".ts,.tsx\"",
-    "build:dev": "RNDS_LOG_LEVEL=debug run-s build:ts:dev types",
-    "build:ts:dev": "webpack --mode production --config=webpack/dev.js",
+    "build:dev": "BABEL_ENV=cjs RNDS_LOG_LEVEL=debug webpack --config=webpack/dev.js",
```

Comment:
I've kept `build:dev` for producing a development bundle, and removed the `dev` and ``build:ts:dev` scripts.

Reply:
Sounds good to me!